pkgs/servers/etcd/3.5/update.sh

```diff
@@ -1,11 +1,27 @@
 #!/usr/bin/env nix-shell
-#!nix-shell -i bash -p curl gnugrep gnused jq nix-prefetch
+#!nix-shell -i bash -p curl gnugrep gnused jq nurl
 
 set -x -eu -o pipefail
 
+MAJOR_VERSION=3
+MINOR_VERSION=5
+
+WORKDIR=$(mktemp -d)
+trap "rm -rf ${WORKDIR}" EXIT
+
+LATEST_TAG_RAWFILE=${WORKDIR}/latest_tag.json
+curl --silent -f ${GITHUB_TOKEN:+-u ":$GITHUB_TOKEN"} \
+    https://api.github.com/repos/etcd-io/etcd/releases > ${LATEST_TAG_RAWFILE}
+
+LATEST_TAG_NAME=$(cat ${LATEST_TAG_RAWFILE} | \
+    jq -r 'map(select(.prerelease == false))' | \
+    jq 'map(.tag_name)' | \
+    grep -v -e rc | tail -n +2 | head -n -1 | sed 's|[", ]||g' | \
+    sort -rV | grep -E "^v${MAJOR_VERSION}\.${MINOR_VERSION}\." | head -n1)
```

Comment:
This is already done later in the file right?

Reply:
Updated from:

>      sort -rV | grep -E "^v${MAJOR_VERSION}\.${MINOR_VERSION}\." | head -n1)

To:

> sort -rV | grep -E "^v${ETCD_VERSION_MAJOR_MINOR}\." | head -n1)

Reply:
I have removed the duplication.